dev/fattest.simplicity/src/com/ibm/ws/fat/util/ShrinkWrapSharedServer.java

```diff
@@ -193,6 +194,8 @@ protected void before() {
             for (String path : archivesAndPaths.get(archive)) {
                 try {
                     ShrinkHelper.exportToServer(getLibertyServer(), path, archive);
+                    String appName = archive.getName().substring(0, archive.getName().length() - 4); //remove the file type suffix
+                    getLibertyServer().addInstalledAppForValidation(appName);
```

Comment:
Can you be sure that an app deployed here is meant to start?

This seems like something only the test would know.